src/Entity/SecurityAdvisory.php

```diff
@@ -70,6 +71,9 @@ class SecurityAdvisory
     #[ORM\Column(type: 'string', nullable: true)]
     private string|null $composerRepository = null;
 
+    #[ORM\Column(nullable: true)]
+    private Severity|null $severity = null;
```

Comment:
It may make sense to use a numeric score value instead of enum. This will make it easier to compare or sort.